### PR TITLE
ui: small fixes to DB Console charts shown for secondary tenants

### PR DESCRIPTION
**Pull request by abarganier**
https://github.com/cockroachdb/cockroach/pull/97995 updated the
DB Console to filter out KV-specific charts from the metrics page
when viewing DB Console as a secondary application tenant.

The PR missed a couple small details. This patch cleans those
up with the following:

- Removes KV latency charts for app tenants
- Adds a single storage graph for app tenants showing livebytes
- Removes the "Capacity" chart on the Overview dashboard for app
  tenants

Release note: none

Epic: https://cockroachlabs.atlassian.net/browse/CRDB-12100

NB: Please only review the final commit. 1st commit is being reviewed separately @ https://github.com/cockroachdb/cockroach/pull/99860

### CODE REVIEW

**Comment by abarganier** [comment]
> Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.crdb.dev/cockroachdb/cockroach?pullRequest=99947) before we can accept your contribution.
1 out of 2 committers have signed the CLA.

This is due to my new machine setup. The issue has been resolved in the original PR https://github.com/cockroachdb/cockroach/pull/99860, so this will go away once that's merged & I rebase.

**Comment by abarganier** [comment]
Okay, now that https://github.com/cockroachdb/cockroach/pull/99860 has been merged, this is ready for a review 🎉

**Review by dhartunian — APPROVED** [review]
:lgtm: 

> Reviewed 4 of 4 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/99947#-:-NSRMtkr1bhquXjbPtiz:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @abarganier)

**Comment by abarganier** [comment]
TFTR! 

bors r=dhartunian

**Comment by rickystewart** [comment]
bors r=dhartunian

**Pull request merged**
